rails-time-track/app/controllers/api/users_controller.rb

```diff
@@ -1,13 +1,50 @@
 module Api
   class Api::UsersController < ApplicationController
+    before_action :set_user, only: [:show, :update, :destroy]
```

Comment:
What about `:create`
I'm not current. authorization isn't implemented yet, right?

Reply:
`set user` is a private action that allows get an existing user and then run only `show, update & destroy` actions. In case of `create` is not convenient.